modelbaker/iliwrapper/ili2dbargs.py

```diff
@@ -68,8 +75,16 @@ def _get_db_args(configuration, hide_password=False):
                         "--dbpwd",
                         configuration.base_configuration.super_pg_password,
                     ]
-                else:
+                elif configuration.dbpwd:
                     db_args += ["--dbpwd", configuration.dbpwd]
+                elif configuration.dbauthid:
+                    # Operations like Export can work with authconf
+                    # and with no superuser login
+                    from ..utils.db_utils import get_authconfig_map
```

Comment:
why you import it here and not on top? Is this kind of better practice?

Reply:
In the other edited file I was getting circular import errors when importing it from the top.

Somehow I missed to move it to the top in the `ili2dbargs.py`. Fixed in the latest commit, thanks!